conans/model/build_info.py

```diff
@@ -35,6 +35,20 @@ def __copy__(self):
         return the_copy
 
 
+def merge_dicts(dict1, dict2):
```

Comment:
Seems the same as the above ``add_dicts``, and also could be shorter and a bit more idiomatic.

Reply:
actually not, this is appending the items in a different order